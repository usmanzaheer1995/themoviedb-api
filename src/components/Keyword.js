import React from 'react';

const Keyword = (props) => {
    return (
        <li>
            {props.keyword}
        </li>
    );
}

export { Keyword as default }