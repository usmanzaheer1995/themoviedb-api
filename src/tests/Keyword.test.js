// Have test case implemented (simple snapshot testing is sufficient)

import { shallow } from 'enzyme';
import React from 'react';
import toJson from 'enzyme-to-json';

import Keyword from '../components/Keyword';

test('should render Keyword correctly', () => {
    const wrapper = shallow(<Keyword keyword={"Test keyword"} />);
    expect(wrapper).toMatchSnapshot();
});