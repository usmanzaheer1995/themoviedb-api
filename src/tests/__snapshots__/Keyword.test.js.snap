// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Keyword correctly 1`] = `
<li>
  Test keyword
</li>
`;
